lib/indexer/index_documents.rb

```diff
@@ -25,7 +25,7 @@ class IndexDocuments
     }
 
     def process(message)
-      $stdout.puts "Processing message: #{message.payload.inspect}"
+      $stdout.puts "Processing message: #{message.payload.slice("content_id", "base_path", "document_type", "title", "update_type", "publishing_app")}"
```

Comment:
Can drop the `$stdout`, just extra line noise.

Also maybe multiline this with a heredoc, for readability?